CONTRIBUTING.md

```diff
@@ -30,11 +30,12 @@ Contributions via pull requests are much appreciated. Before sending us a pull r
 To send us a pull request, please:
 
 1. Fork the repository.
-2. Modify the source; please focus on the specific change you are contributing. If you also reformat all the code, it will be hard for us to focus on your change.
-3. Ensure local tests pass.
-4. Commit to your fork using clear commit messages.
-5. Send us a pull request, answering any default questions in the pull request interface.
-6. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
+2. Install [pre-commit](https://pre-commit.com/) through your preferred mechanism (eg: `pip install`), then run `pre-commit install` **inside this repo**. This will ensure that a collection of [git hooks](./.pre-commit-config.yaml) run on each of your commits, which help minimize your diffs and avoid nitpicking.
```

Comment:
shouldn't these all be 1. numbered, in keeping with your later commits?

Reply:
I think the thought process on my end was that if we decided to drop the mdformat hook, because the introduction and application of it comes further in the series, then it would drop cleanly.

Reply:
ok; fair enough.